arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/predef.kt

```diff
@@ -1,14 +1,40 @@
 package arrow.core
 
+import arrow.syntax.function.curried
```

Comment:
I think the `syntax` namespace is no longer needed. Can these functions live directly in the `arrow.core` namespace?

Reply:
It can but then we need to update the deprcation on `0.12.0`, I can take care of that first in another PR then :)